src/components/SwapForm/SwapModal/ConfirmSwapModalContent.tsx

```diff
@@ -53,6 +75,12 @@ const ConfirmSwapModalContent: React.FC<Props> = ({
 
   const shouldDisableConfirmButton = isBuildingRoute || !!errorWhileBuildRoute
 
+  const priceImpactFromBuild =
```

Comment:
Needs a TODO here. We'll implement a common function to calculate price impact later

Reply:
no need for now. more things need to do in next week